templates/module/helfi_tpr/tpr-unit--wide-teaser.html.twig

```diff
@@ -1,17 +1,25 @@
+{%
+  set content_liftup_classes = [
+    'content-liftup',
+    not unit_published ? 'content-liftup--unpublished',
```

Comment:
This variable could be more generic. Like `not content_entity_published ? 'content-liftup--unpublished',`

Reply:
Made it generic 🦖